### PR TITLE
Improve user page

### LINKED ISSUE #164: [Authentication] User Profile
As a user, I would like to have a profile page so that I am able to visually check my progress.

### PR DESCRIPTION
**Pull request by jaytula**
The `ScoreGraph` component was refactored so it could be re-used in the displaying the Quiz Results. The Quiz Results Page is improved by a bunch of styling.

Closes #164

### CODE REVIEW

**Review by ZumDeWald — APPROVED** [review]
Pulled this down and it looks great!

**Review by cMikolai — APPROVED** [review]
Aye, looking good

File: components/quizSingle/ScoreGraph.tsx [hunk 1]
```diff
@@ -1,8 +1,11 @@
-import { GraphSVG, GraphPathBG, GraphPath, GraphText } from "./styles";
```
Comment (Soreg) [inline components/quizSingle/ScoreGraph.tsx:1]: Why are these moved out of /styles?

The point here is to have a seperation of concerns. We should try avoiding having too much CSS in our functional .tsx styles. This is why we have the seperated styles file.

File: pages/profile.tsx [hunk 2]
```diff
@@ -6,43 +6,88 @@ import { getUserData } from "~/db/users";
...
-              <div>{result.secondsToComplete}</div>
-              <div>{(100* result.numberCorrect / result.totalQuestions).toFixed(2)}%</div>
-            </QuizResultListItem>
-          ))}
-        </ul>
+        <Headline>Your Quiz Results</Headline>
+
+        <QuizResultList>
+          {userData?.quizResults?.map(result => {
+            const percentCorrect =
```
Comment (Soreg) [inline pages/profile.tsx:68]: For readability, I prefer to have this in one line. Though that's nit-picking

File: components/quizSingle/styles.ts [hunk 3]
```diff
@@ -280,21 +280,6 @@ export const TileNumber = styled.span`
   right: 15px;
 `;
 
-export const GraphSVG = styled.svg`
```
Comment (Soreg) [inline components/quizSingle/styles.ts:283]: (Related to first comment)
The style file _already exists_. Is there a specific reason to move styles out from here?

**Review by Soreg — COMMENTED** [review]
Declined until somments has been resolved
